### ISSUE TITLE
FileNotFoundError on first run.

### ISSUE THREAD

**Issue by Animesh-Ghosh** [sharanya02/Text-file-to-handwritten-pdf-file#6]
After cloning, creating a virtual environment using `venv` and installing requirements, running the script for the first time gives an FileNotFoundError:

```pwsh
python txttohandwriting.py
FileNotFoundError: [Errno 2] No such file or directory: 'final_output.pdf'
```

I worked around it by creating an empty file by the name `final_output.pdf` and then running the script again, which seems to work.

It would be better if file creation was handled by the script, or some mention of this issue in the `README.md` was made.

**Comment by projjal1** [comment]
Yeah, I have done a pull request which fixes this issue.  Thanks for noticing this bug.

**Comment by elvismartis** [comment]
HI
I guess here is a fix to be appended to txttohandwriting.py
`#Now I am opening each image
filehandle = open('final_output.pdf', 'w+')
for PNG_FILE in imagelist:
    rgba = Image.open(PNG_FILE)
    rgb = Image.new('RGB', rgba.size, (255, 255, 255))  # white background
    rgb.paste(rgba, mask=rgba.split()[3])  # paste using alpha channel as mask
    rgb.save('final_output.pdf',
             append=True)  #Now save multiple images in same pdf file
filehandle.close()`

**Comment by Animesh-Ghosh** [comment]
@elvismartis please use code blocks to properly format code in conversations. Please see how GitHub markdown works.

**Comment by elvismartis** [comment]
@Animesh-Ghosh I apologise. I  Will add it as code blocks.

**Comment by Animesh-Ghosh** [comment]
@sharanya02 why was this issue closed? Has the issue been resolved? #8 is still open so I thought the issue is still not fixed.
